geopandas/tools/sjoin.py

```diff
@@ -24,9 +45,9 @@ def sjoin(left_df, right_df, how='inner', op='intersects',
         Suffix to apply to overlapping column names (left GeoDataFrame).
     rsuffix : string, default 'right'
         Suffix to apply to overlapping column names (right GeoDataFrame).
+    tree_idx : rtree.index for right GeoDataFrame, if None, will be created.
```

Comment:
Can you put the explanation on a next line (see how it is done for the other kwargs). Typically it should have the form of (following the numpy docstring standard)

```
kwarg : type
    Explanation
```